R/to_rmd.R

```diff
@@ -56,67 +58,69 @@ to_rmd.default <- function(block, output_dir, ...) {
 #' @keywords internal
 .to_rmd.Reporter <- function(block,
                              output_dir,
-                             yaml_header,
-                             global_knitr = getOption("teal.reporter.global_knitr"),
-                             include_chunk_output,
+                             rmd_yaml_args,
                              ...) {
-  blocks <- block$get_blocks()
+  checkmate::assert_list(rmd_yaml_args, names = "named")
+  tc <- as.teal_card(block$get_blocks())
+  metadata(tc) <- rmd_yaml_args
+  .to_rmd(tc, output_dir = output_dir, ...)
```

Comment:
Give the chance here to use overwritten methods)

```suggestion
  to_rmd(tc, output_dir = output_dir, ...)
```